javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,41 @@
+const JsUserInput = document.getElementById("UserInput");
+const JsButton = document.getElementById("Button");
+const JsNoOfgifs = document.getElementById("NoOfgifs");
+const JsDispGif = document.getElementById("DispGif");
+let url;
+
+function getGiphy()
+{
+    fetch(url)
```

Comment:
In general, it is a good idea to add code for error handling. For example in cases when the requests fails.
`fetch(url)
.then(response) => {}
.catch((error) => {
  console.error('Error:', error);
});
`